projects/storefrontlib/cms-components/navigation/search-box/search-box-component.service.ts

```diff
@@ -31,6 +31,9 @@ export class SearchBoxComponentService {
   chosenWord = new ReplaySubject<string>();
   sharedEvent = new ReplaySubject<KeyboardEvent>();
 
+  private enableRecentSearches: boolean = false;
```

Comment:
`major:` let's make this field `protected` for easier extensibility and customisation